HW04/aleksei_zhuchkov/lib/notification.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Notification
+  attr_reader :title, :description
+
+  def initialize(title = 'None', description = 'None')
+    @title = title
+    @description = description
+  end
+
+  def print_notifications(notifications)
```

Comment:
Again, I am very against any mixing of business logic and information output in the code.
If you want, you can simply return the information you want and display it in your app.rb however you like. 
In the second option, you can create a Printer class that will do this.
(I recommend the first option)

Reply:
Hi, I have done it. See my commit https://github.com/labs-ruby/homeworks-2022/pull/96/commits/60daa14b13bcf23ecb46a62da61e662f4206d2dd
tnx